aten/src/ATen/native/cpu/BinaryOpsKernel.cpp

```diff
@@ -488,7 +488,7 @@ void logical_xor_kernel(TensorIterator& iter) {
 }
 
 void rshift_kernel(TensorIteratorBase& iter) {
-  AT_DISPATCH_INTEGRAL_TYPES(iter.dtype(), "rshift_cpu", [&]() {
+  AT_DISPATCH_INTEGRAL_TYPES_AND(kUInt64, iter.dtype(), "rshift_cpu", [&]() {
```

Comment:
I guess this needs to be changed to explicitly return 0 for `unsigned` as no bits causes `Kright shift count >= width of type [␛[K-Werror=shift-count-overflow␛` to fire?